$NetBSD: patch-Makefile,v 1.1 2015/06/13 19:31:34 leot1990 Exp $

o Get rid of the -D install(1) option.
o Properly create the BINDIR like all other directories used.

--- Makefile.orig	2015-06-13 15:29:55.000000000 +0000
+++ Makefile
@@ -19,7 +19,8 @@ clean:
 	@$(MAKE) $(MFLAGS) -C tests clean
 
 install: $(TARGET) $(DOCDIR)/$(MAN1)
-	install -D -m 755 $(SRCDIR)/$(TARGET) $(DESTDIR)$(BINDIR)/$(TARGET)
+	install -d $(DESTDIR)$(BINDIR)
+	install -m 755 $(SRCDIR)/$(TARGET) $(DESTDIR)$(BINDIR)/$(TARGET)
 	install -d $(DESTDIR)$(EXAMPLEDIR)
 	cp -r examples/* $(DESTDIR)$(EXAMPLEDIR)
 	install -d $(DESTDIR)$(MANDIR)/man1
